cmake/toolchains/arm64.cmake

```diff
@@ -0,0 +1,19 @@
+# Fizzy: A fast WebAssembly interpreter
+# Copyright 2020 The Fizzy Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+# The toolchain file for arm64 cross-compilation.
+# It selects default version of GCC targeting aarch64-linux-gnu, tested with GCC 10.
+
+set(CMAKE_SYSTEM_PROCESSOR arm64)
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)
+set(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
```

Comment:
Shouldn't this file be named `aarch64-linux` perhaps?

Reply:
Can be `arm64-linux` if you prefer.

Reply:
I think it is more correct that way.

Reply:
For the other case, I prefer `arm64` over `aarch64`. It is more informative. Kinda similar issue to `x86-64` vs `amd64`.